oak_functions/loader/src/server.rs

```diff
@@ -440,14 +485,14 @@ fn oak_functions_resolve_func(
         }
     };
 
-    if &sig != signature {
+    if signature != &expected_signature {
         return Err(wasmi::Error::Instantiation(format!(
-            "Export `{}` doesn't match expected type {:?}",
-            field_name, signature
+            "Export `{}` doesn't match expected type {:?}, expected: {:?}",
```

Comment:
```suggestion
            "Export `{}` doesn't match expected signature: got {:?}, expected: {:?}",
```
nit!

Reply:
Done in https://github.com/project-oak/oak/pull/2010